src/config.rs

```diff
@@ -714,7 +715,7 @@ impl HookBuilder {
 
     /// Create a `PanicHook` and `EyreHook` from this `HookBuilder`.
     /// This can be used if you want to combine these handlers with other handlers.
-    pub fn into_hooks(self) -> (PanicHook, EyreHook) {
+    pub fn into_hooks(self) -> Result<(PanicHook, EyreHook), crate::eyre::Report> {
```

Comment:
I wasn't paying attention and didn't realize this was introducing a breaking change. I'd prefer to avoid a major version bump, can we rename this to `try_into_hooks` and re-add `into_hooks` with the original signature and panic behavior?

Reply:
sure thing